jwt/token.go

```diff
@@ -25,7 +25,8 @@ type UnpackVerifyFunc func(s string) (*Token, error)
 
 // Token wrapper
 type Token struct {
-	Claims `bson:"inline"`
+	Claims   `bson:"inline"`
+	TenantID string `json:"tenant_id" bson:"tenant_id"`
```

Comment:
yes. this is the best way, and I went both ways. with all the tests back and forth. what is in the claims is one thing, but (according to the units) mender.tenant is not always present, so I left it duplicated.

Reply:
Another reason I don't want to introduce this field is that I would prefer omitting the document migration for this collection since the tokens are time bound, and existing tokens are cached in Redis. Besides, clients should always attempt to reauthorize once a token becomes invalid.